internal/vppinit/apiparams/const.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024 Nordix Foundation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package apiparams provides parsing factilty for configuration parameters
+// file for vpp init
+package apiparams
+
+const (
+	confFilename = "/var/lib/networkservicemesh/vppapi-hostint-args.yaml"
```

Comment:
Is it possible to move this parameter to the NSM config? 
I think it will be easier to see that vppinit can be configured

Reply:
Yes, eventually that was my initial approach.  I can define this parameter as NSM config parameter and

1/ Pass it to init (Execute function) either changing argument list or using options pattern - each will change the function prototype and made reading/understanding the code harder. For example in main.go the call of this init function is complex already:
`vppinit.Must(cfg.VppInit.Execute(ctx, vppConn, cfg.TunnelIP)),`

Or
2/ Get the parameter value by `os.Getenv()` where I want to use the path name in `apiparams.go: getConfig(...)`

Reply:
@glazychev-art Going forward with 2nd alternative, detailed in comment above.